docs/OTAnalytics/usage-ui.md

```diff
@@ -95,140 +106,189 @@ To add videos, use the *Add...* button, and to remove them from the configuratio
 When you click a video in the configuration bar, the first image of the video is displayed as the background image.
 This allows you to check whether all videos have been selected correctly.
 
-### Save to configuration file (otconfig file)
+### Save and open the project
 
-Once all the required videos have been added and all the flows have been created, the project is saved.
+Once you added all required tracks and videos and created all sections and flows, the project can be saved.
 
 1. Click on the *Save as...* button in the Project section of the configuration bar.
-1. Select a suitable filename in the file browser and save it in the same folder as the videos. 
+2. Select a suitable filename in the file browser and save it in the same folder as the videos.
+
+You can save the project in two ways:
+
+An `.otconfig` file stores all information necessary to reload the project in OTAnalytics.
+This is the default option.
+
+An `.otflow` file stores only information about sections and flows.
+Hence, it can be reused / imported into another project with different videos of the same measurement site.
 
 !!! tip
     We recommend regularly saving the progress of the project while it is still being processed.
     This prevents possible loss of data.
     The *Save* button automatically saves the file to the last selected location.
     If it is colored in orange, changes have been made.
 
-    We also recomment to store all project related files (videos, otdet, ottrk and otconfig) in the same folder.
+    We also recomment to store all project related files (videos, `.otdet`, `.ottrk` and `.otconfig`) in the same folder.
 
-!!! info
-    An otconfig files stores all information necessary to reload the project in OTAnalytics. 
-    An otflow file stores only information about the sections and flows. 
-    Hence, it can be reused / imported into other project with different videos of the same measurement site.
+When you return later, you can reopen the project using the *Open ...* button.
 
-## Definition of the traffic flows to be analyzed
+## Configure traffic analysis
 
+To perform traffic analyses with OTAnalytics, sections and, in many cases, flows are needed.
```

Comment:
The majority of users doing traffic analysis need flows. Therefore, I would skip the `in many cases`, because it is more like a default that you need the flows.

Reply:
I kind of disagree - but I don't insist :)